main.js

```diff
@@ -137,10 +139,13 @@
 
   function checkGuesses() {
     numberOfGuesses += 2;
-    guessOneDisplay.innerText = challengerGuesses[0].value;
-    guessTwoDisplay.innerText = challengerGuesses[1].value;
-    checkGuess1();
-    checkGuess2();
+    var guess1 = checkGuess1();
+    var guess2 = checkGuess2();
+    if (guess1 == randomNumber) {
```

Comment:
We did this as a fix to a winner reseting the number before we checked the next guess